### ISSUE TITLE
Add 'share' link with read-only API key in dashboard 

### ISSUE THREAD

**Issue by glynhudson** [emoncms/dashboard#205]
This is a really neat way to share / access a dashboard without logging in. Could a 'share'button be added to the dashboard list with this URL auto generated for a particular dash? https://community.openenergymonitor.org/t/accessing-dashboards-using-read-only-apikey/1538

**Comment by pb66** [comment]
I agree an easy way to get a share link like this would be very useful.

2 points though, firstly if/when the dashboard is set public could there also be a button for the public url? 

And secondly, use of read apikey like this should be done with extreme caution. Once you share your read key to anyone with any emoncms knowledge, they have access to all you raw feed and input data too, not just what you chose to display in your  dashboards. So perhaps we need a 3rd "view apikey" that can only give access to dashboards, (saved) graphs and apps would be better.

That view key could have the ability to be changed so you can revoke access.

Ideally a system where multiple keys could Grant access to different collections would be even better, but that is far more complex. For now it would be good to dish out a key that does'nt reveal all and can be changed easily.
